tests/components/sensor/test_yr.py

```diff
@@ -53,19 +49,15 @@ def test_custom_setup(self, betamax_session):
                    return_value=betamax_session):
             with patch('homeassistant.components.sensor.yr.dt_util.utcnow',
                        return_value=now):
-                assert sensor.setup(self.hass, {
-                    'sensor': {
-                        'platform': 'yr',
-                        'elevation': 0,
-                        'monitored_conditions': {
-                            'pressure',
-                            'windDirection',
-                            'humidity',
-                            'fog',
-                            'windSpeed'
-                        }
-                    }
-                })
+                assert _setup_component(self.hass, 'sensor', {
+                                        'sensor': {'platform': 'yr',
+                                                   'elevation': 0,
+                                                   'monitored_conditions': {
+                                                       'pressure',
+                                                       'windDirection',
+                                                       'humidity',
+                                                       'fog',
+                                                       'windSpeed'}}})
```

Comment:
Why is monitored conditions here not a list? According to the configuration sample in the docs the monitored_conditions is specified as a list so the yaml parser would return `[ 'pressure', 'windDirection', ...]`.